### PR TITLE
button

### PR DESCRIPTION
**Pull request by Mandhularajitha**
Added button

### CODE REVIEW

File: button.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.float {
+    border: none;
+    color: rgb(59, 39, 240);
+    padding: 8px 16px;
+    font-size: 30px;
+    cursor: pointer;
+    border-radius: 60px;
+    box-shadow: rgb(211, 192, 192);
```
Comment (manismk) [inline button.css:88]: Use rem for Responsiveness

File: button.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    background-color: rgb(39, 243, 12);
+    color: black;
+    border: 2px solid;
+}
+
+.button2 {
+    background-color: rgb(39, 235, 235);
+    color: black;
+    border: 2px solid #0f0f0e;
+    border-radius: 8%;
```
Comment (manismk) [inline button.css:24]: Be consistent in color values. Either use rgb or use hex code. Avoid using direct color names

File: button.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.button2:hover {
+    background-color: #f552ea;
+    color: rgb(55, 13, 241);
+}
+
+.button3 {
```
Comment (manismk) [inline button.css:32]: Instead of button1, button2 . Naming can be given as button-primary or button-blue.

File: button.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    border: 2px solid #0f0f0f;
+}
+
+h1 {
+    text-align: center;
+}
+
+body {
+    background-color: #EFF6FF;
+}
```
Comment (manismk) [inline button.css:98]: Avoid direct tag styling

**Review by manismk — CHANGES_REQUESTED** [review]

**Review by neetukumari4858 — APPROVED** [review]

**Review by Sindhu983 — APPROVED** [review]

**Pull request merged**